### PR TITLE
Implement welcome page designs on homepage

### PR DESCRIPTION
**Pull request by tdooner**
## Ticket

Resolves FFS-1373.

## Changes

* This implements the "/welcome" styles on the homepage
* I've made a second attempt at moving the footer down so it's not so
  big for short pages (like the homepage).
* Since the homepage no longer has the "Start flow manually" button,
  I've removed the functionality allowing starting a flow without an
  invitation.
* Adds a "message_html" option to slim alert in case you want to do some
  bold text or something in one of those alerts

## Context for reviewers

N/A

## Testing

Screenshots below.

### CODE REVIEW

**Comment by tdooner** [comment]
image

Expired session:
image

**Comment by tdooner** [comment]
FYI @carmenrosalop - Homepage built and ready for acceptance. See screenshots above ^

File: app/config/locales/en.yml [hunk 1]
```diff
@@ -230,6 +229,12 @@ en:
         total_payments_desc: This is the total income from your job(s) before taxes, benefits and deductions were taken out of your paycheck.
       update:
         consent_to_authorize_warning: You must check the legal agreement checkbox to proceed.
+  pages:
+    home:
+      description_1: The SNAP Income Pilot is a new tool designed to help you connect your income details from your employer or payroll provider directly to your benefits agency. We're currently testing this tool to ensure it works effectively.
+      description_2: Please note this pilot is currently available only to SNAP participants in New York City and Massachusetts.
+      description_3_html: '<p>To participate, you can request an invitation from your benefits agency:</p><ul><li>Massachusetts applicants: Contact the <a href="https://www.mass.gov/guides/how-to-contact-dta" target="_blank">Department of Transitional Assistance (DTA)</a></li><li>New York City applicants: Contact the <a href="https://www.nyc.gov/site/hra/help/snap-application-frequently-asked-questions.page">Human Resources Administration (HRA)</a></li></ul>'
```
Comment (GeorgeCodes19) [inline app/config/locales/en.yml:236]: Is this naming convention or is rails doing something special here?

**Review by GeorgeCodes19 — APPROVED** [review]

File: app/config/locales/en.yml (hunk 1 above)
Comment (tdooner) [reply app/config/locales/en.yml:236]: Nope - nothing special. I just didn't want to think of three names to describe the three separate paragraphs.

**Review by carmenrosalop — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**